test/storage.test.js

```diff
@@ -3,6 +3,7 @@ const {
   getFeed,
   getFeeds,
   getFeedsCount,
+  removeFeed,
```

Comment:
Can you add a test for `removePost` too, since that got added as well?

Reply:
Added